### PR TITLE
Fix(revit): regression on switching ElementId to UniqueId

### PR DESCRIPTION
**Pull request by oguzhankoral**
It was a regression on switch from ElementId to UniqueId, since we send report items as UniqueId, it can't parse them directly to ElementId

### CODE REVIEW

File: Connectors/Revit/Speckle.Connectors.RevitShared/ElementIdHelper.cs [hunk 1]
```diff
@@ -5,13 +5,14 @@ namespace Speckle.Connectors.RevitShared;
 
 public static class ElementIdHelper
 {
-  public static ElementId Parse(string idStr)
+  public static ElementId GetElementIdByUniqueId(Document doc, string uniqueId)
```
Comment (didimitrie) [inline Connectors/Revit/Speckle.Connectors.RevitShared/ElementIdHelper.cs:8]: I'd name this `GetElementIdFromUniqueId`, as we're transmuting an unique id into an element id, and in my head sounds clearer

File: Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs [hunk 2]
```diff
@@ -130,10 +130,14 @@ public async Task Send(string modelCardId)
...
 
+      var activeUIDoc =
+        RevitContext.UIApplication?.ActiveUIDocument
+        ?? throw new SpeckleException("Unable to retrieve active UI document");
+
       List<ElementId> revitObjects = modelCard
         .SendFilter.NotNull()
         .GetObjectIds()
-        .Select(ElementIdHelper.Parse)
+        .Select(uid => ElementIdHelper.GetElementIdByUniqueId(activeUIDoc.Document, uid))
```
Comment (didimitrie) [inline Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs:140]: i think there's some unecessary bouncing around here, from unique ids to element ids: in the root object builder, we can just use [GetElement(string)](https://www.revitapidocs.com/2019/b5a1473c-21c4-6c29-f1cf-26822c955260.htm) - where the string is the unique id.

**Review by didimitrie — CHANGES_REQUESTED** [review]
briefly tested, and it works as advertised; minor request only. I might do PR

File: Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs (hunk 2 above)
Comment (oguzhankoral) [reply Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs:140]: The helper function already do it but yes, simplified as you suggested

**Review by didimitrie — APPROVED** [review]

**Pull request merged**